File: Assignment_2/Assignment_2.c

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;
    while(iNo > iCnt)
    {
        printf(" * ");
        iNo--;
    }
    printf("\n");
}
int main()
{
    int iValue = 0 ;

    printf("Enter Number to Print * : ");
    scanf("%d",&iValue);
    Display(iValue);

    return 0 ;
}

File: Assignment__11/Assignment_java/Assignment_2.java
import java.util.*;

class Pattern 
{
    public void Display(int iNo)
    {
        int iCnt = 0;
       
        for(iCnt = iNo ; iCnt >= 1 ; iCnt--)
        {
            System.out.print(iCnt+" # ");
        }
        System.out.println();
    }
}
class Assignment_2
{


    public static void main(String A[])
    {   
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number : ");
        int iValue = sobj.nextInt();

        Pattern pobj = new Pattern();

        pobj.Display(iValue);
    }
}